mlir-tensorrt/python/bindings/Runtime/RuntimePyBind.cpp

```diff
@@ -371,11 +371,11 @@ createMemRefViewFromDLPack(PyRuntimeClient &client, py::capsule capsule) {
     s = mtrtMemRefCreateExternal(client, addressSpace, bytesPerElement * 8,
                                  reinterpret_cast<uintptr_t>(data), offset,
                                  rank, shape, strides, device, elementType,
-                                 &result);
+                                 &result, true /*assertCanonicalStrides*/);
```

Comment:
Why are we forcing this to true here? It should be a a parameter to the function.

Reply:
I have updated the PR to add Python API assertion check option.